apply-revapi-suggestions.java

```diff
@@ -0,0 +1,159 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
I suggest you put this in `.jbang/RevapiUpdate.java` to make it easier to edit/use in IDE's directly.